test/plugin/customParseFormat.test.js

```diff
@@ -72,3 +75,47 @@ it('fails with an invalid format', () => {
   expect(dayjs(input, format).format().toLowerCase())
     .toBe(moment(input, format).format().toLowerCase())
 })
+
+it('parse month from string', () => {
+  const input = '2018 February 03'
+  const format = 'YYYY MMMM DD'
+  expect(dayjs(input, format).valueOf()).toBe(moment(input, format).valueOf())
+})
+
+it('parse month from short string', () => {
+  const input = '2018 Feb 03'
+  const format = 'YYYY MMM DD'
+  expect(dayjs(input, format).valueOf()).toBe(moment(input, format).valueOf())
+})
+
+it('parse month from short string in other locales', () => {
+  const input = '2018 May 03'
+  const inputUk = '2018 трав 03'
+  const format = 'YYYY MMM DD'
+  dayjs.locale(uk)
+  expect(dayjs(inputUk, format).valueOf()).toBe(moment(input, format).valueOf())
```

Comment:
this is not a good test .

we should compare with 
```
moment.locale('uk')
```

Reply:
done.
I also write test with passing locale in config (btw, it is undocumented for now)